installer/windows/StorjValidateUpgradeDlg.wxs

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
+
+
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <Fragment>
+        <UI>
+            <Dialog Id="StorjValidateUpgradeDlg" Width="370" Height="270" Title="!(loc.VerifyReadyDlg_Title)" TrackDiskSpace="yes">
+                <Control Id="UpgradeTitle" Type="Text" X="15" Y="15" Width="300" Height="15" Transparent="yes" NoPrefix="yes" Text="{\WixUI_Font_Bigger}Ready to upgrade [ProductName]" />
+                <Control Id="UpgradeDescription" Type="Text" X="25" Y="70" Width="320" Height="100"  Text="The Setup Wizard will upgrade your the Storj V3 Storage Node installation. This operation will not modify your storage node's configuration." />
```

Comment:
Can we include the version we are upgrading to in this text message?

For example:
> The Setup Wizard will upgrade your Storj V3 Storage Node installation to version 0.25.2. 

Note: remove "the" between "your" and "Storj V3".